core/services/feeds/service.go

```diff
@@ -0,0 +1,45 @@
+package feeds
+
+import (
+	"context"
+)
+
+//go:generate mockery --name Service --output ./mocks/ --case=underscore
+
+type Service interface {
+	CountManagerServices() (int64, error)
```

Comment:
These functions wrap the ORM implementation so we don't need to expose the ORM directly to the application.